app/managers.py

```diff
@@ -0,0 +1,42 @@
+import sqlite3
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("app/cinema.sqlite")
+        self.table_name = "actors"
+
+    def create(self, first_name: str, last_name: str) -> None:
+        self._connection.execute(
+            f"INSERT INTO {self.table_name} (first_name, last_name) "
+            f" VALUES (?, ?)",
```

Comment:
Using f-strings for SQL queries can lead to SQL injection vulnerabilities. It's safer to use parameterized queries as you have done with the placeholders (?, ?).